### ISSUE TITLE
Creamlinux doesnt seem to be working

### ISSUE THREAD

**Issue by Quark04** [anticitizn/creamlinux#21]
Followed the steps and put my files under the CK3 directory (/mnt/Gamedrive/SteamLibrary/steamapps/common/Crusader Kings III) and launched it with a more recent .ini file with the correct launch option, my only DLC has a red triangle but no others appear, ran the logging step and got this error.

**Comment by anticitizn** [comment]
Hello, thanks for posting. Could you please post the full log? I believe the error was caused by the script being unable to find `libsteam_api.so`. It appears to work fine for others, so I'm curious what's causing it to fail in your situation.

**Comment by NL-TCH** [comment]
> Hello, thanks for posting. Could you please post the full log? I believe the error was caused by the script being unable to find `libsteam_api.so`. It appears to work fine for others, so I'm curious what's causing it to fail in your situation.

Yes, #31 is related i think, the libsteam_api.so cannot be found (when using GEproton 7.55)

**Comment by Red-Skuller** [comment]
I also couln't start it by modifying the launch options but could start it manually from the game folder with `konsole --hold -e sh ./cream.sh /home/user/.local/share/Steam/ubuntu12_32/reaper SteamLaunch AppId=1158310 -- /home/user/.local/share/Steam/ubuntu12_32/steam-launch-wrapper -- '/home/user/.local/share/Steam/steamapps/common/SteamLinuxRuntime_soldier'/_v2-entry-point --verb=waitforexitandrun -- '/home/user/.local/share/Steam/steamapps/common/SteamLinuxRuntime'/scout-on-soldier-entry-point-v2 --  '/home/user/.local/share/Steam/steamapps/common/Crusader Kings III/launcher/dowser' --pdxlDisableHardwareSettings -nakama -simultaneouschunks 2\0
`
The dlcs weren't unlocked
The output on the konsole was
`ERROR: ld.so: object '/tmp/lib64Creamlinux.so' from LD_PRELOAD cannot be preloaded (wrong ELF class: ELFCLASS64): ignored.
ERROR: ld.so: object '/tmp/libsteam_api.so' from LD_PRELOAD cannot be preloaded (wrong ELF class: ELFCLASS64): ignored.
ERROR: ld.so: object '/tmp/lib32Creamlinux.so' from LD_PRELOAD cannot be preloaded (wrong ELF class: ELFCLASS32): ignored.
ERROR: ld.so: object '/tmp/lib32Creamlinux.so' from LD_PRELOAD cannot be preloaded (wrong ELF class: ELFCLASS32): ignored.
ERROR: ld.so: object '/tmp/lib32Creamlinux.so' from LD_PRELOAD cannot be preloaded (wrong ELF class: ELFCLASS32): ignored.
ERROR: ld.so: object '/tmp/lib32Creamlinux.so' from LD_PRELOAD cannot be preloaded (wrong ELF class: ELFCLASS32): ignored.
ERROR: ld.so: object '/tmp/lib32Creamlinux.so' from LD_PRELOAD cannot be preloaded (wrong ELF class: ELFCLASS32): ignored.
ERROR: ld.so: object '/tmp/lib32Creamlinux.so' from LD_PRELOAD cannot be preloaded (wrong ELF class: ELFCLASS32): ignored.
ERROR: ld.so: object '/tmp/lib32Creamlinux.so' from LD_PRELOAD cannot be preloaded (wrong ELF class: ELFCLASS32): ignored.
`

**Comment by anticitizn** [comment]
Okay, I think there's several unrelated errors here. Proton obviously doesn't work with creamlinux. 
Regarding the `./cream.sh: line 25: fg: no job control`, please try this version of the launch script and see if it still occurs: 
https://github.com/user-attachments/files/17017800/cream.sh.zip

Maybe something is also wrong with the paths in the launch script, funny that it works with an absolute one. Thanks for debugging thoroughly @Red-Skuller
